src/lib/containers/SubsectionRowRenderer.tsx

```diff
@@ -62,7 +63,7 @@ const SubsectionRowRenderer: React.FunctionComponent<SubsectionRowRendererProps>
         },
       }
 
-      const queryResultBundle = await SynapseClient.getFullQueryTableResults(
+      const queryResultBundle = await SynapseClient.getQueryTableResults(
```

Comment:
Since you're not getting the full results, optionally you could now use (useGetQueryResultBundle)[https://github.com/Sage-Bionetworks/Synapse-React-Client/blob/62ab80316421af5d3126739e5e81b9fca5745156/src/lib/utils/hooks/SynapseAPI/useGetQueryResultBundle.ts#L7] if it works with the rest of this component.